checkdoc/main.go

```diff
@@ -4,27 +4,24 @@
 // you may not use this file except in compliance with the License.
 // You may obtain a copy of the License at
 //
-//       http://www.apache.org/licenses/LICENSE-2.0
+//	http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
nit: this should be reverted

```suggestion
//     http://www.apache.org/licenses/LICENSE-2.0
````

Reply:
Looks like this revert is what is causing the `ci / lint` test to fail. Running `make precommit` updates the spacing back to what is was before I reverted.

Reply:
Looks like you were able to get it right :) Thank you